tools/ci_build/github/azure-pipelines/mac-coreml-ci-pipeline.yml

```diff
@@ -32,7 +32,7 @@ jobs:
   workspace:
     clean: all
   pool:
-    vmImage: 'macOS-13'
+    vmImage: 'macOS-latest'
```

Comment:
macos-latest is currently macos-12 (probably an inidcator that macos-13 is not stable enough). Using this string for now, so that we will get macos-13 once it is stable enough. We will have to manually upgrade the xcode version though.